### ISSUE TITLE
"Split biomes, and modify parameters" tests failing, not sure why

### ISSUE THREAD

**Issue by bpbond** [JGCRI/hector#456]
Commenting out these lines in `test_biome.R` as part of #455 . They are failing, and it's not clear to me whether this is a problem or some unexpected (but correct) model behavior.

```
  expect_lt(
    subset(r_warm_data, biome == "permafrost" &
      variable == "detritus_c" &
      year == 2100)[["value"]],
    subset(r_biome_data, biome == "permafrost" &
      variable == "detritus_c" &
      year == 2100)[["value"]]
  )
  expect_lt(
    subset(r_warm_data, biome == "permafrost" &
      variable == "soil_c" &
      year == 2100)[["value"]],
    subset(r_biome_data, biome == "permafrost" &
      variable == "soil_c" &
      year == 2100)[["value"]]
  )
```

**Comment by leeyap** [comment]
Also not sure why this isn't working. 

From what I can see, we want `r_warm_data` to have `det_c` and `soil_c` values _lower_ than `r_biome_data` because the warming factor for the permafrost biome within `r_warm_data` is set at 3.0. When I tried to retrieve the warming factor for the `r_biome` just to check, I got this error:

> Error in sendmessage(core, GETDATA(), v, dates, NA, "") : 
  sendmessage: msg:  	Assertion failed: Biome 'global' missing from biome list. Hit this error while trying to retrieve variable: 'warmingfactor'.

I assumed that by default it would be set to one, but when looking at the results the `r_warm_data` values for `det_c` and `soil_c` were always higher than `r_biome_data` and were always increasing. I extended the run years from 2000 - 2100 to 2300. In 2238, `soil_c` begins to decrease in `r_warm_data` but begins to decrease in 2103 in the `r_biome_data`. `det_c` begins to decrease in 2117 for the `r_biome_data` but continuously increases in `r_warm_data`.

Not sure if maybe `det_c` and `soil_c` should actually be lower in the `r_biome_data` situation than in `r_warm_data`...
